std/stdio.d

```diff
@@ -1522,6 +1524,139 @@ for every line.
     }
 
 /**
+Creates a file with a randomly generated name and returns an open File to it.
+
+There is nothing special about the file created other than the fact that its
+name is randomly generated, and it defaults to being in the temp directory on
+the system. It will only be deleted if explicitly deleted or if it's in the
+temp directory, and the temp directory is cleared out (as happens on system
+startup or shutdown on some systems).
+
+The file is created with R/W permissions and opened with $(D "w+b"). On POSIX
+systems, the permissions are restricted to the current user, though the
+effective permissions are modified by the process' umask in the usual way.
+
+Params:
+    prefix = Prefix for the generated file name.
+    suffix = Suffix for the generated file name (which also provides a way to
+             give the file an extension).
+    dir    = Directory of the temporary file. Defaults to the result of
+             $(XREF file, tempDir).
+  */
+    static File tempFile(string prefix = null, string suffix = null)
```

Comment:
Well, that's ugly, since the overload without `dir` is the short one. I'd have hoped that the compiler would be smart enough to not give a warning when one of the documented items matched the documentation - though honestly, even having warnings for documentation like that is a bit off when you consider cases like when a struct and function share documentation as sometimes occurs when a function returns a specific struct. So, while I can certainly see how such warnings could be useful, I question that they actually make sense ultimately.

Regardless, as ugly as it may be, I can rearrange the functions to avoid the warning.